pytest_fixtures/component/satellite_auth.py

```diff
@@ -269,6 +271,19 @@ def enroll_configure_rhsso_external_auth(module_target_sat):
         'yum -y --disableplugin=foreman-protector install '
         'mod_auth_openidc keycloak-httpd-client-install'
     )
+    if settings.server.version.rhel_version == '8':
+        module_target_sat.execute(
+            f'dnf -y install {settings.repos.rhel8_os.appstream}Packages/{PYTHON_LXML}'
+        )
+    module_target_sat.execute(
+        f'openssl s_client -connect {settings.rhsso.host_name} -showcerts </dev/null 2>/dev/null| '
+        f'sed "/BEGIN CERTIFICATE/,/END CERTIFICATE/!d" > {CERT_PATH}/rh-sso.crt'
+    )
+    module_target_sat.execute(
+        f'sshpass -p "{settings.rhsso.rhsso_password}" scp -o "StrictHostKeyChecking no" '
```

Comment:
I am not really a fan of `scp`. 😕 Can't you use `remote_copy` as we do in `robottelo.hosts.Capsule`?
https://github.com/SatelliteQE/robottelo/blob/203557349cfa32af5800922b40f4e2634967280c/robottelo/hosts.py#L1256-L1258

Reply:
The example you are giving which is more passing satellite and capsule object. RHSSO is currently part of any object. Hence I tried to avoid this. This is not a single file copy there are 3 .crt files.

Reply:
I am OK with it. I will leave this discussion open if someone else wants to comment.